docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,731 @@
+          {
+            Port = 9190
+            Authorization
+            {
+              Default = ServiceAdministrator
+              componentExists = authenticated
+              getComponents = authenticated
+              hostExists = authenticated
+              getHosts = authenticated
+              installationExists = authenticated
+              getInstallations = authenticated
+              updateLog = Operator
+            }
+          }
+          SystemAdministrator
+          {
+            Port = 9162
+            Authorization
+            {
+              Default = ServiceAdministrator
+              storeHostInfo = Operator
+            }
+          }
+        }
+        URLs
+        {
+          ComponentMonitoring = dips://dirac-tuto:9190/Framework/ComponentMonitoring
+          SystemAdministrator = dips://dirac-tuto:9162/Framework/SystemAdministrator
+        }
+        FailoverURLs
+        {
+        }
+        Databases
+        {
+          InstalledComponentsDB
+          {
+            DBName = InstalledComponentsDB
+            Host = localhost
+            Port = 3306
+          }
+        }
+      }
+    }
+  }
+
+
+This configuration will be used for example by Services in order to:
+
+* know their configuration (for example the `ComponentMonitoring` Service will use everything under `Systems/Framework/Production/Services/ComponentMonitoring` )
+* Identify host and persons (`Registry` section)
+
+Or by clients to get the URLs of given services (for example `ComponentMonitoring = dips://dirac-tuto:9190/Framework/ComponentMonitoring`)
+
+Since this configuration is given as a whole to every client, you understand why no database credentials are in this file. Services and Agents running on the machine will have their configuration as a merge of what is served by the Configuration service and the `/opt/dirac/etc/dirac.cfg`, and thus have access to these private information.
+
+The file `/opt/dirac/bashrc` is to be sourced whenever you want to use the server installation.
+
+Client installation
+===================
+
+Now we will create another linux account `diracuser` and another installation to be used as client
+
+--------------------
+Setup client session
+--------------------
+
+This section has to be ran as `root`
+
+First, create an account, and add in its `~/.globus/` directory the user certificate you created earlier::
+
+  adduser -s /bin/bash -d /home/diracuser diracuser
+  passwd diracuser
+  mkdir ~diracuser/.globus/
+  cp /opt/dirac/user/client.pem ~diracuser/.globus/usercert.pem
+  cp /opt/dirac/user/client.key ~diracuser/.globus/userkey.pem
+  chown -R diracuser:diracuser ~diracuser/.globus/
+
+
+--------------------
+Install DIRAC client
+--------------------
+
+This section has to be ran as `diracuser`
+
+We will do the installation in the `~/DIRAC` directory. For a client, the configuration is really minimal, so we will just install the code and its dependencies.
```

Comment:
the `~/DIRAC` directory is IMHO a bad name. It will then have another `DIRAC` directory inside it. Call it differently.

Reply:
What do you propose ? `DiracInstallation` ?

Reply:
It's possible.

Reply:
OK I'll change to that